lighthouse-core/audits/dobetterweb/no-vulnerable-libraries.js

```diff
@@ -0,0 +1,126 @@
+/**
+ * @license
+ * Copyright 2017 Google Inc. All rights reserved.
```

Comment:
we have short license headers now... copy one from one of the other files?

Reply:
All set.